backtester/README.md

```diff
@@ -109,7 +109,7 @@ Creating strategies requires programming skills. [Here](/backtester/eventhandler
 
```

Comment:
Low priority comment related to backtester:
Please update `tpl.gohtml`:
`	<link rel="icon" href="https://raw.githubusercontent.com/thrasher-corp/gocryptotrader/a1a667bab9150e611dc04bad43fa49457171936a/web/src/assets/images/gctlogo-notext.svg" />`